### PR TITLE
Bump libraries and updated mixin usage

### PR DESCRIPTION
**Pull request by FlorianMichael**

### CODE REVIEW

**Comment by TheKodeToad** [comment]
llamalad pointed out to me that @ModifyArgs would be more compatible in some places, if you're interested.

**Comment by FlorianMichael** [comment]
Yeah, I also updated Gradle, should work now

File: src/main/java/io/toadlabs/numeralping/mixin/ServerEntryMixin.java [hunk 1]
```diff
@@ -1,46 +1,46 @@
...
 	// hide the tooltip if it's redundant
-	@Redirect(method = "render", at = @At(value = "INVOKE", target = "Lnet/minecraft/client/gui/screen/multiplayer/MultiplayerScreen;setMultiplayerScreenTooltip(Ljava/util/List;)V"), require = 0)
-	public void setTooltip(MultiplayerScreen screen, List<Text> tooltip) {
-		if (tooltip != null && tooltip.size() == 1 && NumeralConfig.instance().serverList
-				&& tooltip.get(0).getContent() instanceof TranslatableTextContent translatable
-				&& translatable.getKey().equals("multiplayer.status.ping")) {
-			tooltip = null;
+	@ModifyArgs(method = "render", at = @At(value = "INVOKE", target = "Lnet/minecraft/client/gui/screen/multiplayer/MultiplayerScreen;setMultiplayerScreenTooltip(Ljava/util/List;)V"), require = 0)
+	public void setTooltip(Args args) {
+		@SuppressWarnings("unchecked") final var tooltip = (List<Text>) args.get(0);
```
Comment (TheKodeToad) [inline src/main/java/io/toadlabs/numeralping/mixin/ServerEntryMixin.java:40]:
```suggestion
		var tooltip = (List<Text>) args.get(0);
```

**Review by TheKodeToad — APPROVED** [review]

File: src/main/java/io/toadlabs/numeralping/mixin/ServerEntryMixin.java (hunk 1 above)
Comment (FlorianMichael) [reply src/main/java/io/toadlabs/numeralping/mixin/ServerEntryMixin.java]: Done

Comment (TheKodeToad) [reply src/main/java/io/toadlabs/numeralping/mixin/ServerEntryMixin.java]: I don't really want a lot of annotations :P
It's just a harmless warning

**Pull request merged**